playroom/preview-tools.css.ts

```diff
@@ -96,7 +96,7 @@ export const controlsTransitionClasses = {
     }),
 };
 
-export const floatingButtonTopTransitionClasses = controlsTransitionClasses;
+export const floatingButtonTopTransitionClasses = {...controlsTransitionClasses};
```

Comment:
I don't know why this fixes the issue. Maybe some weird things are happening with VE